.circleci/config.yml

```diff
@@ -54,22 +61,86 @@ orbs:
               name: Publish distributions
               command: |
                 pipenv run twine upload dist/*
+      integration-tests:
+        machine:
+          image: ubuntu-2004:202107-02
+        environment:
+          K8S_VERSION: v1.22.0
+          MINIKUBE_VERSION: v1.22.0
+          CHANGE_MINIKUBE_NONE_USER: true
+          MINIKUBE_WANTREPORTERRORPROMPT: false
+          MINIKUBE_WANTUPDATENOTIFICATION: false
+          RUNTIME_SOCK: unix:/var/run/dockershim.sock
+        steps:
+          - checkout
+          - run:
+              name: Install pip dependencies
+              command: |
+                pip3 install pipenv
+                pipenv install --dev --deploy --python=$(which python3)
+          - run:
+              name: Setup system prereqs
+              command: |
+                sudo apt-get update
+                sudo apt-get install -y conntrack
+          - run:
+              name: System info
+              command: |
+                uname -a
+                cat /etc/os-release
+                lscpu
+                free -mt
+          - run:
+              name: Install minikube
+              command: |
+                curl -sSfLo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64
+                chmod +x minikube
+                sudo mv minikube /usr/local/bin/
+          - run:
+              name: Launch minikube
+              command: |
+                sudo -E minikube start \
+                  --vm-driver=none \
+                  --kubernetes-version=${K8S_VERSION} \
+                  --wait=true
+
+                # Allow `circleci` user access
+                sudo -E chmod 777 "${RUNTIME_SOCK/unix:/}"
+          - run:
+              name: Run tests
+              command: pipenv run pytest --junitxml=./tests/reports/junit-test-report.xml
```

Comment:
This is the same command as the unit tests. Is https://github.com/dominodatalab/container-runtime-interface-api/pull/6/files#diff-447902925392013bc18e0262800d36f08679774aa2a192c109dddb1c01370a71R9 the only _integration test_? Do you want to separate unit from integration tests entirely instead of running unit tests twice (essentially)?

Reply:
It's just so fast to run the unit tests that I feel like it doesn't hurt. It also doesn't require someone to understand that there are two sets of distinct tests. I added a section to the README to indicate they're combined for now.